chain/client/src/client.rs

```diff
@@ -1655,13 +1655,15 @@ impl Client {
                 info!(target: "client", "not producing a chunk");
             }
         }
-        if let Err(err) = self.shadow_validate_block_chunks(&block) {
-            tracing::error!(
-                target: "client",
-                ?err,
-                block_hash = ?block.hash(),
-                "block chunks shadow validation failed"
-            );
+        if provenance != Provenance::SYNC {
```

Comment:
can you add comment about why you are checking SYNC (the same reasoning in the PR description)

Reply:
Added